app/views/templates/news_article.html.haml

```diff
@@ -3,26 +3,30 @@
 
 - breadcrumb :public_news_article, node
 
-%div.list-horizontal
-  = render partial: 'templates/related_sections', locals: {node: node}
-
 %h1
   =node.name
 
 %div.abstract
   %p
     =node.short_summary
     - if node.summary
-      != markdown_content node.summary
+      .summary
+        != markdown_content node.summary
 
 %ul.announcement-info
   - if node.release_date
-    %li Published date: #{l(node.release_date, format: :news)}
+    %li
+      %span
+        Date published
+      #{l(node.release_date, format: :news)}
   %li
-    Published by:
-    %a= link_to(node.section.name, public_node_path(node.section.home_node))
-
+    %span
+      Published by
+    %a= link_to(node.section.name, section_news_articles_path(section: @section.home_node.slug))
 
 = render partial: 'templates/content_body', object: node.content_body
 
 =@hierarchy
+
+-#%div.list-horizontal
+  = render partial: 'templates/related_sections', locals: {node: node}
```

Comment:
@ltankey We aren't sure how to define related articles yet. We are leaving out related news articles from the platform for now because there needs to be a conversation with the platform people about what is possible for an MVP. One thing that came up in usability testing was that if the related articles weren't meaningfully related then it would be super annoying. Could just be manually entered in the article editor, could be rails magic etc